code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -143,45 +143,56 @@
 ///mob/living/carbon/monkey/diona/New()
 //Moved to it's duplicate declaration modules\mob\living\carbon\monkey\diona.dm
 
+/mob/living/carbon/monkey/movement_delay()
+	var/tally = 0
+	if(reagents)
+		if(reagents.has_reagent("hyperzine")) return -1
+
+		if(reagents.has_reagent("nuka_cola")) return -1
+
+	var/health_deficiency = (100 - health)
+	if(health_deficiency >= 45) tally += (health_deficiency / 25)
+
+	if (bodytemperature < 283.222)
+		tally += (283.222 - bodytemperature) / 10 * 1.75
+
+	if(istype(loc,/turf/simulated/floor))
+		var/turf/simulated/floor/T = loc
+
+		if(T.material=="phazon")
```

Comment:
This is shitcode copypasted from humans, I didn't even touch it, git diff thinks I did because I moved a couple of procs down into the file. Someone was up to a rework of this a while ago, outta scope for me